src/components/TaxCalculator.js

```
@@ -1,119 +1,287 @@
 
-    if (isNaN(salaryAmount) || isNaN(newSalaryAmount)) {
-      alert("Please enter valid salary amounts.");
-      return;
-    }
+      // Default to current date if no endDate is provided
+
+      const salaryAmount = parseFloat(salary);
+      const salaryAmountPerDay = salaryAmount / 30;
+
+      if (isNaN(salaryAmount)) {
+        alert("Please enter valid salary amounts.");
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)